build/build-release.yml

```diff
@@ -1,14 +1,17 @@
 variables:
   buildConfiguration: Release
 
+pr: none
+
 # Batch builds
 trigger:
   branches:
     include:
       - main
   paths:
     include:
-      - src/WebJobs.Extensions.OpenAI*/*
+      - src/Directory.Build.props
+      - src/Functions.Worker.Extensions.OpenAI/AssemblyInfo.cs
```

Comment:
if we're setting `pr: none`, then are these `trigger` settings even needed?

Reply:
for CI?

Reply:
How does this ADO pipeline get triggered? I was under the impression that it was manually triggered since you configured `pr: none` above. If it's manually triggered, then why do we need to have path includes? I thought that was only for automatically triggered pipelines?

Reply:
ADO has bit different trigger settings than github, the keyword trigger is used for continuous integration triggers (here after merge to main branch and from included file paths only)